example_rules/example_new_term.yaml

```diff
@@ -1,5 +1,5 @@
-# Alert when a login event is detected for a never before seen combination of (username, computer)
-# In this example, "login" logs contain which user has logged into what computer
+# Alert when a login event is detected for user "admin" never before seen ip
+# In this example, "login" logs contain which user has logged in from what IP
```

Comment:
This prose uses "ip" and "IP" interchangeably. Please use just one capitalization.